pkg/controller/manager.go

```diff
@@ -139,6 +143,12 @@ func (cm *controllerManager) injectInto(i interface{}) error {
 	if _, err := inject.DoInformers(cm.informers, i); err != nil {
```

Comment:
I guess we need a fix here.
We expect to have it inject whichever params that `i` implement, rather than return directly once an error occur